tests/Tester.py

```diff
@@ -0,0 +1,42 @@
+import csv
+
+from apis import WikipediaApi
+from src import HeuristicTester
+from src.heuristics import Heuristics
+
+test_file_path = "tests.txt"
+results_file_path = "results.tsv"
```

Comment:
I think we should add a timestamp or something to the filename so that we can keep old test results

Reply:
Also if we can write it to something like `/results/{timestamp}.tsv` and not in the source package that'd be nice 👍